OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -154,30 +154,39 @@ public func isMethodDELETE() -> OHHTTPStubsTestBlock {
 /**
  * Matcher for testing an `NSURLRequest`'s **scheme**.
  *
+ * e.g. the scheme part is `https` in `https://api.example.com/signin`
+ *
  * - Parameter scheme: The scheme to match
  *
  * - Returns: a matcher (OHHTTPStubsTestBlock) that succeeds only if the request
  *            has the given scheme
  */
 public func isScheme(_ scheme: String) -> OHHTTPStubsTestBlock {
+  precondition(!scheme.contains("://"), "The scheme part of an URL never contains '://'. Only use strings like 'https' for this value, and not things like 'https://'")
```

Comment:
Seems like `contains` was not available in Swift 2.3 🙄 Maybe add an `extension String` in the `#if !swift(>=3.0)` block at the top of this file to add support for it? https://github.com/AliSoftware/OHHTTPStubs/blob/5956336837ccb8d483119fede8cdee6590be8485/OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift#L30-L55